add ca warning note for basic + catchup minor fixes in master (#161)
mspnp/microservices-reference-implementation: deployment.md

## Prerequisites

- Azure subscription

>   > Important: The user initiating the deployment process must have access to the `Microsoft.Authorization/roleAssignments/write` permission. For more information, please refer to [the Container Insights doc](https://docs.microsoft.com/en-us/azure/azure-monitor/insights/container-insights-troubleshoot#authorization-error-during-onboarding-or-update-operation)

Reviewer: ```suggestion
  > Important: The user initiating the deployment process must have access to the **Microsoft.Authorization/roleAssignments/write** permission. For more information, see [the Container Insights doc](https://docs.microsoft.com/en-us/azure/azure-monitor/insights/container-insights-troubleshoot#authorization-error-during-onboarding-or-update-operation)
```

## Deploy the ingress controller

> :warning: WARNING
>

> > Certificates created by these scripts MUST NOT be used for production. They

Reviewer: ```suggestion
> Do not use the certificates created by these scripts for production. The
```

> > are provided for demonstration purposes.

Reviewer: ```suggestion
> certificates are provided for demonstration purposes only.
```

> > When productizing against CA Certificates, you'll need to use your own

Reviewer: ```suggestion
> For your production workload, use your ```
